scripts/mysqld_safe.sh

```diff
@@ -258,6 +258,16 @@ add_mysqld_ld_preload() {
   lib_to_add="$1"
   log_notice "Adding '$lib_to_add' to LD_PRELOAD for mysqld"
 
+  # Check if the library is in the reduced number of standard system directories
+  case "$lib_to_add" in
+    /usr/lib64* | /usr/lib/x86_64-linux-gnu* | /usr/lib* | ${MY_BASEDIR_VERSION}/lib*)
```

Comment:
You're correct, but it even seems more correct for me to add "/" on the paths to really make it possible to only use /usr/lib/.. and not something like /usr/library/... although /usr is writeable by root only so it should be fine but anyway.